docs/oss-oncall-log.md

```diff
@@ -0,0 +1,13 @@
+# OSS oncall log
+
+"Oh yeah, I remember having to deal with that a while back... what was it again?"
+
+- Issue: https://github.com/kubernetes/test-infra/issues/8703
```

Comment:
Instead of just linking the issue here, can we give it a more descriptive link text eg: [Velodrome is out-of-date](https://github.com/kubernetes/test-infra/issues/8703)?

Reply:
I feel like that's additional friction to what (after some thinking overnight) I am concerned is too manual a way to flag issues, but yeah let's give it a shot.